### DISCUSSION TITLE
Where is the env file please? I can't turn off the Access Control mode

### DISCUSSION THREAD

**Discussion by le-khang** [zanllp/infinite-image-browsing#236] [Q&A] [answered]
Thank you for creating this helpful extension, but for the past few days I've been unable to use it. It said
"To ensure data security, you are currently running in access control mode, which only allows access to authorized folders. You can adjust access permissions by editing the .env file." 
but I don't know where is that file. I came across a file named env.d.ts, but it contains nothing except this line: /// reference types="vite/client" /. Could you please guide me on how to resolve this issue?

**Comment by zanllp** [comment, marked as answer]
I understand that you're having trouble finding the .env configuration file. However, the .env.example file that I provided serves as a template for you to create your own .env file.

To use this template, simply make a copy of the .env.example file and rename it to .env. Then, you can fill in the necessary values for your specific use case.


<img width="943" alt="image" src="https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/25872019/ffb2bd9c-c62f-481a-85af-978a282a7d8b">

**Reply by le-khang** [reply]
Thank you for your quick response but I have deleted the extension then re-installed it and still there is no .evn file
![2023-06-14 21_18_55-Window](https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/35536989/f600bf21-cbc3-459f-8761-8118745e86b5)
![2023-06-14 21_18_08-Window](https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/35536989/8009e8a5-5b7f-44f4-983f-840d016368a4)
![2023-06-14 21_17_27-Window](https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/35536989/812b7c1f-35d8-4780-9c28-a93a00fd43d3)

**Reply by zanllp** [reply]
This file is in the root directory of this extension, not in the "vue" folder. It is located one level above the folder shown in your screenshot.

**Reply by zanllp** [reply]
Perhaps it's hidden by your file browser, you can manually create one.

https://github.com/zanllp/sd-webui-infinite-image-browsing/blob/main/.env.example

**Reply by le-khang** [reply]
I tried to create one and upload it to JupyterLab (I'm using Runpod, a cloud GPU service), but I can't find them anywhere. I googled it and found out that JupyterLab doesn't serve .gitignore or any other hidden files (which are prefixed with a .) by default, as mentioned in this article: https://matttriano.org/2021-08-13-configuring-jupyterlab-p1/
I'm not a technical person, so I'm not sure how to follow the steps mentioned in the article. Additionally, I'm not even sure if Runpod allows users to edit their Jupyter server.

**Reply by zanllp** [reply]
You can also directly set the environment variable IIB_ACCESS_CONTROL=disable

https://stackoverflow.com/questions/37890898/how-to-set-env-variable-in-jupyter-notebook

**Reply by le-khang** [reply]
![2023-06-16 10_14_08-Window](https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/35536989/8e7647f1-2447-4ae6-a4b6-2c42d497adb1)
I'm not sure if I do it right but it always said "no such job". Even when I enter %env by itself to print out current environmental variables it also said "no such job" :((

**Reply by zanllp** [reply]
`%env` is used inside a cell. I am not sure how you started it specifically, but if you started it using the shell command `./webui-user.sh`, you can try adding environment variables in the `./webui-user.sh` file.
<img width="472" alt="image" src="https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/25872019/eb4e27bb-1e1c-4c66-af5b-7e74fb2ec7b3">

They are all based on the same principle.

**Comment by steinhaug** [comment]
What excactly are theese settings for ? Do I need to add them at all ?I am currently running the as standallone, I do see the .env.example file is there however no clue what the parameters are...

**Reply by zanllp** [reply]
These configurations are mainly related to privacy and security. If you are using standalone mode, all of them will be turned off. If it doesn't affect you, you don't need to worry about them.
